### PR TITLE
fix(amplify-category-analytics): delete pinpoint project in delete

### LINKED ISSUE #2974: Tearing down environment still does not delete pinpoint
Tearing down environment still does not delete pinpoint

**Amplify CLI Version**
4.6.0

**To Reproduce**
- Run `npx react-native init AnalyticsTest`
- Run `amplify init`
- Run `amplify add analytics` with all defaults
- Run `amplify push` 
- Pinpoint console now contains `analyticstest-env` 
- Run `amplify delete`
- ERROR: Pinpoint console still contains `analyticstest-env`

According to a previous, closed issue, this was fixed
https://github.com/aws-amplify/amplify-cli/issues/1278
But it still occurs in amplify cli v.4.6.0

**Desktop**
 - OS: Windows
 - Node 12.9.1

### PR DESCRIPTION
**Pull request by UnleashedMind**
currently the amplify delete command does not delete the pinpoint project, this PR fixes that

fix #2974

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ammarkarachi** [comment]
@UnleashedMind Are you going to add e2e tests for this?

**Comment by ammarkarachi** [comment]
LGTM!

File: packages/amplify-e2e-tests/src/__tests__/delete.test.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ import { addEnvironment, checkoutEnvironment, removeEnvironment } from '../envir
 import { addApiWithoutSchema } from '../categories/api';
 import { addCodegen } from '../codegen/add';
 import * as fs from 'fs-extra';
+import * as pinpointHeler from '../utils/pinpoint';
```
Comment (ammarkarachi) [inline packages/amplify-e2e-tests/src/__tests__/delete.test.ts]: Can you correct the spelling here?

Comment (UnleashedMind) [reply packages/amplify-e2e-tests/src/__tests__/delete.test.ts]: done

**Pull request merged**